File: level_1/문자열_내_마음대로_정렬하기/devSOWON0628.js
function solution(strings, n) {
    let result = strings.sort((a,b)=>{
        if(a[n] > b[n]){
            return 1
        }else if(a[n] < b [n]){
            return -1
        }else{
            if(a.toString(2)>b.toString(2)){
                return 1
            }else{
                return -1
            }
        }
    })
    return result;
}

File: level_2/JadenCase_문자열_만들기/devSOWON0628.java
class Solution {
    public String solution(String s) {
        String[] splited = s.toLowerCase().split(" ");
        StringBuilder result = new StringBuilder();
        
        for(int i = 0 ; i < splited.length ; i++){
            String word = splited[i].trim();
            if(word.length()<1){
                result.append(" ");
                continue;
            }
            result.append(word.substring(0, 1).toUpperCase());
            result.append(word.substring(1,word.length()));
            result.append(" ");
        }
        if(!" ".equals(s.substring(s.length()-1, s.length()))){
            result.deleteCharAt(result.lastIndexOf(" "));
        }
        return result.toString();
    }
}
